hasher-matcher-actioner/hmalib/lambdas/actions/reactioner.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import json
+
+from hmalib.common.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def lambda_handler(event, context):
+    """
+    This is the main entry point for reacting to ThreatExchange. The action evaluator
+    sends a reaction message by way of the reactions queue and here's where they're
+    popped off and dealt with.
+    """
+    for sqs_record in event["Records"]:
+        # TODO research max # sqs records / lambda_handler invocation
+        sqs_record_body = json.loads(sqs_record["body"])
+        logger.info(f"sqs_record_body = {sqs_record_body}")
+        # TODO next PR will include implementation of the ReactionMessage class
```

Comment:
My fear is that the logic here is going to look almost identical to the logic in the action_performer. Unclear to me if that will become an issue

Reply:
@jeberl  Yup we'll have to see. One big difference is what gets persisted at the end of the day. In the case of actions, you'll be writing a row somewhere that's keyed off of the ID of the matched content. For reactions, it'll be something persisted by "banked content id" (indicator id).